src/renderer/db.ts

```diff
@@ -1,8 +1,25 @@
 import Dexie from 'dexie';
+import { keybindingsDefault } from './hooks/useKeybindings';
 
 export const db = new Dexie('query-master');
 
 db.version(1).stores({
   database_config: 'id',
   database_tabs: 'id',
+  keybindings: 'id',
```

Comment:
Maybe we don't want to use keybinding database like that. We can create `setting` database instead and with key "keybinding" to access keybinding information